File: output/gemini-3.1-pro-preview/few/cpp/Relógio.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    long long h, m, s, t;
    if (cin >> h >> m >> s >> t) {
        long long total_seconds = h * 3600 + m * 60 + s + t;
        total_seconds %= 86400;
        
        long long new_h = total_seconds / 3600;
        total_seconds %= 3600;
        long long new_m = total_seconds / 60;
        long long new_s = total_seconds % 60;
        
        cout << new_h << "\n" << new_m << "\n" << new_s << "\n";
    }

    return 0;
}

File: output/gemini-3.1-pro-preview/few/python/Relógio.py
import sys

input_data = sys.stdin.read().split()
if not input_data:
    exit()

H = int(input_data[0])
M = int(input_data[1])
S = int(input_data[2])
T = int(input_data[3])

total_s = H * 3600 + M * 60 + S + T

new_S = total_s % 60
total_m = total_s // 60
new_M = total_m % 60
total_h = total_m // 60
new_H = total_h % 24

print(new_H)
print(new_M)
print(new_S)